skbio/io/format/tests/test_genbank.py

```diff
@@ -0,0 +1,224 @@
+from __future__ import absolute_import, division, print_function
+from future.builtins import map, range, zip
+import six
+
+import io
+from unittest import TestCase, main
+from functools import partial
+
+
+class ReaderTests(TestCase):
+    pass
+
+gbk_example_dna = '''LOCUS       SCU49845     5028 bp    DNA             PLN       21-JUN-1999
```

Comment:
These are large enough they warrant their own files in the associated `data` directory.